Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -761,3 +751,15 @@ private extension ChatMessageLayoutOptions {
         !intersection(.metadata).isEmpty
     }
 }
+
+extension ChatMessageLayoutOptions {
```

Comment:
I have extracted the corners logic to the `layoutOptions` extension so it can be reused at the gallery view. 
Ig you have an idea for a better place, let me know!

Reply:
I was just thinking if it makes sense to make this public or not... But I guess no need of it, it's just convenient extension... IMHO it's good in here but maybe someone has some other opinion 🙃